fdbrpc/FlowTransport.actor.cpp

```diff
@@ -1435,6 +1438,10 @@ NetworkAddress FlowTransport::getLocalAddress() const {
 	return self->localAddresses.address;
 }
 
+const std::unordered_map<NetworkAddress, Reference<struct Peer>>& FlowTransport::getAllPeers() const {
```

Comment:
I think we don't need the `struct` keyword here?

Reply:
Good catch. Done.